plugins/notifications-backend/src/service/router.ts

```diff
@@ -389,7 +439,7 @@ export async function createRouter(
   };
 
   // Add new notification
-  router.post('/', async (req, res) => {
+  router.post('/notification', async (req, res) => {
```

Comment:
`/notifications` imo, just always sticking to plural for REST resources

Reply:
Sure thing